front-end/src/hooks/useCreateTodo.ts

```diff
@@ -0,0 +1,25 @@
+import { useAppDispatch } from "../app/hooks";
+import { addTodoItemAction } from "../features/todo/todoSlice";
+import { TodoAPI } from "../services";
+import { TodoType } from "../types";
+
+export default function useCreateTodo() {
+    const dispatch = useAppDispatch();
+
+    const addHandler = async (todoItem: TodoType) => {
+        TodoAPI.createTodo(todoItem);
```

Comment:
Why this line here, `TodoAPI.createTodo(todoItem);` being called twice?

Reply:
yeah usko remove krta hun